chariot-core/src/player/choices.rs

```diff
@@ -43,7 +43,7 @@ lazy_static! {
         (Stat::CarSpin, GLOBAL_CONFIG.car_spin),
         (Stat::MaxCarSpeed, GLOBAL_CONFIG.max_car_speed),
         (Stat::MaxCarSpin, GLOBAL_CONFIG.max_car_spin),
-        (Stat::Mass, 10.0),
+        (Stat::Mass, 0.01),
```

Comment:
not even god understands how this number relates to the real force of gravity

Reply:
also, since we have a global config stat, we should reference it
```suggestion
        (Stat::Mass, GLOBAL_CONFIG.gravity_coefficient),
```